plug.vim

```diff
@@ -2050,14 +2050,14 @@ function! s:git_validate(spec, check_branch)
         let [output, shellerror] = s:system(printf(
               \ 'git rev-list --count --left-right HEAD...origin/%s',
               \ a:spec.branch), a:spec.dir)
-        let [ahead, behind] = split(s:lastline(output), '\t')
+        let [ahead; behind] = split(s:lastline(output), '\t')
         if !shellerror && ahead
-          if behind
+          if !shellerror && ahead && len(behind) == 1
```

Comment:
Sorry if I wasn't being clear. What I meant is to change line 2054 and 2055 as follows:

```vim
if !shellerror && ahead && len(behind) == 1
  if behind[0]
```

1. First if statement checks if the output of the command conforms to the expected format
2. Second if statement checks if `behind[0]` is a non-zero value.